redaxo/src/core/lib/packages/package.php

```diff
@@ -246,12 +246,11 @@ public function loadProperties()
             $this->propertiesLoaded = true;
             return;
         }
-        if (
-            isset($cache[$id]) &&
-            (!rex::isBackend() || !rex::getConsole() && (!rex::getUser() || !rex::getUser()->isAdmin()) || $cache[$id]['timestamp'] >= filemtime($file))
+        
+        $isCached = isset($cache[$id]);
+        $isBackendAdmin = rex::isBackend() && rex::getUser() && rex::getUser()->isAdmin();
+        if ( !$isCached || rex::getConsole() || $isBackendAdmin && $cache[$id]['timestamp'] < filemtime($file)
```

Comment:
Die Logik ist jetzt aber eine andere als vorher.
Vorher wurde beispielsweise in der Console der Cache schon genutzt, wenn der Timestamp-Check passte.
Jetzt wird der Cache niemals genutzt in der Console.

Der Gedanke bei der vorherigen Version war: Im Frontend und bei Non-Admins nicht mal die Timestamps prüfen, sondern immer dem Cache vertrauen, wenn vorhanden.

Reply:
War noch eine Klammer zu wenig. Jetzt sollte es passen